### PR TITLE
Roles in Data Science Industry

### PR DESCRIPTION
**Pull request by lalith1403**
#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.
   
This fixes #395 

#### Grammar Checks
- [x] Have you spellchecked and grammar checked your article? - **Tool:** [Grammarly](https://grammarly.com)
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical Checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [ ] Have you used Repl.it for your code snippets?

### Contribution Guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**

### CODE REVIEW

**Comment by hectorkambow** [comment]
plagiarism check done

File: articles/roles-data-science/index.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+   - Embedded Systems: [Raspberry-pi](https://www.raspberrypi.org/), [Beaglebone Black](https://beagleboard.org/black),[Nvidia Jetson](https://developer.nvidia.com/buy-jetson), [Movidius](https://www.intel.com/content/www/us/en/products/processors/movidius-vpu.html)
+   - Programming environments: [Anaconda](https://www.anaconda.com/), [H20.ai](https://www.h2o.ai/)
+   - Programming Languages: [Python](https://www.python.org/), C/C++
+   - Commonly used libraries: [Tensorflow](https://www.tensorflow.org/), [PyTorch](https://pytorch.org/), [Tensorflow Lite](https://www.tensorflow.org/lite), [Scikit-learn](https://scikit-learn.org/)
+2. Machine Learning Scientist Tools:
+   - Machine Learning Frameworks: [Tensorflow](https://www.tensorflow.org/), [PyTorch](https://pytorch.org/), [MxNet](https://mxnet.apache.org/), [Caffe](http://caffe.berkeleyvision.org/)
+   - Parallel and Distributed Programming Frameworks: [CUDA](https://developer.nvidia.com/cuda-downloads), [OpenGL](https://www.opengl.org/)
+3. Data Analysts Tools:
+   - Exploratory Data Analysis:
+   - Databases Management Systems:
```
Comment (hectorkambow) [inline articles/roles-data-science/index.md:113]: I think you may have forgot to include any 'Exploratory Data Analysis' tools? @lalith1403

Comment (lalith1403) [reply articles/roles-data-science/index.md:113]: I had mentioned the same under machine learning engineer tools: Thus might have missed it out... I have added the tools for EDA. Thank you.

**Comment by hectorkambow** [comment]
👋  Good morning @kurt-sectionio 
Wanted to ask if you could give this "Roles in the Data Science Industry" article a tech review when you have a moment before we publish. 
Seemed like a very informative article - learned a lot about what data scientist do. 🚀

**Comment by kurt-sectionio** [comment]
Getting back on track after some leave. Will review tomorrow.

**Comment by hectorkambow** [comment]
> Getting back on track after some leave. Will review tomorrow.

Oh - my bad Kurt - didn't realize you were on leave - thank you v much. 
No rush 👍

**Review by kurt-sectionio — APPROVED** [review]
Looks good. Thank you. A few minor suggestions. Please address these before merging.

I'm a little confused by the heading for Framing The Problem that appears between ML Scientists and Data Analysts. I would suggest excluding that heading and using instead a simple sentence to introduce the importance of problem-framing. Something like: "One way ML scientists bring value is through framing difficult problems so that they can solved and managed effectively and efficiently." 

line 17 space between "intelligence(AI)"

line 35: "multiple changes are possible" may be better than "applicable"

**Pull request merged**